pyproject.toml

```diff
@@ -21,7 +21,44 @@ keywords = [
     "ui",
     "data",
 ]
+dependencies = [
+  "htbuilder",
+  "st-annotated-text",
+  "streamlit-embedcode",
+  "streamlit-keyup"
+]
```

Comment:
I think there's a way to just read from Pipfile. But let's go ahead with that.

Q: Isn't that missing `streamlit`? I guess it's included by transitivity, though...